src/foundry/client/apps/av/cameras.d.ts

```diff
@@ -94,10 +107,9 @@ declare global {
 
     /**
      * Dynamically refresh the state of a single camera view
-     * @param view - The view container div
-     * @internal
+     * @param userId - The ID of the user whose view we want to refresh.
      */
-    protected _refreshView(view: HTMLElement): void;
+    protected _refreshView(userId: string): void;
```

Comment:
weird change lol. I trust that it happened don't get me wrong, just a weird change.